src/WSEvents.cpp

```diff
@@ -911,6 +915,38 @@ void WSEvents::OnSceneItemVisibilityChanged(void* param, calldata_t* data) {
 	instance->broadcastUpdate("SceneItemVisibilityChanged", fields);
 }
 
+/**
+ * An item's transfrom has been changed.
+ *
+ * @return {String} `scene-name` Name of the scene.
+ * @return {String} `item-name` Name of the item in the scene.
+ *
+ * @api events
+ * @name SourceItemChanged
```

Comment:
It would be best to keep scene item event names in the same fashion as others.
```suggestion
 * @name SceneItemTransformChanged
```